app/models/info_request_batch.rb

```diff
@@ -46,6 +46,8 @@ class InfoRequestBatch < ApplicationRecord
 
   strip_attributes only: %i[embargo_duration]
 
+  scope :not_embargoed, -> { where(embargo_duration: nil) }
```

Comment:
Is this always true?, would a better check be to see if any of the batch's info requests have embargo:

```suggestion
  scope :not_embargoed, -> { left_joins(info_requests: :embargo).where(embargoes: { id: nil }) }
```

Reply:
It's how we define `InfoRequestBatch#embargoed?`. The value gets cleared in [`AlaveteliPro::Embargo.expire_publishable`](https://github.com/mysociety/alaveteli/blob/0.42.0.0/app/models/alaveteli_pro/embargo.rb#L103-L105), although it looks like we might _not_ do so when a user clicks "publish now". I'd _thought_ we'd fixed that, so will have to check.

In general I do think we should be able to rely on it; just need to make sure in practice that we can.

Reply:
> it looks like we might _not_ do so when a user clicks "publish now". I'd _thought_ we'd fixed that, so will have to check.
> 
> In general I do think we should be able to rely on it; just need to make sure in practice that we can.

It _does_ get cleared when clicking "publish requests", so I think this is okay as-is.